editor/selectors.js

```diff
@@ -879,13 +880,15 @@ export function getRecentlyUsedBlocks( state ) {
 
 /**
  * Resolves the block usage stats into a list of the most frequently used blocks.
+ * Memoized so we're not generating block lists every time we render the list
+ * in the inserter.
  *
  * @param {Object} state Global application state
  * @return {Array}       List of block type settings
  */
-export function getMostFrequentlyUsedBlocks( state ) {
+export const getMostFrequentlyUsedBlocks = memoize( ( state ) => {
```

Comment:
Should we use `createSelector` helper instead to memoize the selector (we already use it elsewhere). https://github.com/aduth/rememo

Reply:
I was going to suggest something like this, I didn't see we already had something :) Will fix!